src/listen.js

```diff
@@ -45,6 +45,9 @@ module.exports = function(defaultFuncs, api, ctx) {
       case 'read_receipt':
         globalCallback(null, utils.formatReadReceipt(event));
         return true;
+      case 'read':
```

Comment:
Yes that sounds fine.  Please add the same note to your documentation when you add that.